src/components/ServiceInstallation/Tabs/Services.vue

```diff
@@ -150,11 +161,21 @@
       },
       fetchServiceStatus() {
         this.getServices().then((response) => {
+          const servicesToBeRender = []
+
           this.lastFetched = new Date().toLocaleTimeString()
           const requiredServices = response.filter((service) =>
             this.servicesToCheck.includes(service.id)
           )
-          this.listOfServices = requiredServices
+          const requiredServicesTypeDb = response.filter((service) =>
+            this.servicesToCheckBeforeMovingToDashboard.includes(service.id)
+          )
+          if (this.isAvailableTypeDBInitAndSample) {
```

Comment:
I think in this case based on the condition we can filter, instead filter both 

based on condition filter the respective one.

Reply:
Done